client/src/components/DropdownPace/DropdownPace.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+const Select = styled.select`
+  height: 40px;
+  width: 100%;
+  margin-top: 15px;
+  border: solid 2px var(--secondary-color);
+  background: none;
+  color: var(--secondary-color);
+  font-family: "Heiti SC", adobe-heiti-std, sans-serif;
```

Comment:
`font-family` could be controlled by CSS variables like the colors.

Reply:
Thanks